src/status_im/new_group/handlers.cljs

```diff
@@ -91,6 +93,43 @@
       ((after show-chat!))
       ((after start-listen-group!))))
 
+(register-handler :create-new-public-group
+  (after (fn [_ [_ topic]]
+           (dispatch [:navigation-replace :chat topic])))
+  (u/side-effect!
+    (fn [db [_ topic]]
+      (let [exists? (boolean (get-in db [:chats topic]))
+            group   {:chat-id     topic
+                     :name        topic
+                     :color       default-chat-color
+                     :group-chat  true
+                     :public?     true
+                     :is-active   true
+                     :timestamp   (.getTime (js/Date.))}]
```

Comment:
There is a function `timestamp` in `status-im.utils.random` which does the same.
But:
1) I am not sure that `status-im.utils.random` ns is a great place for this function;
2) There are probably other calls to js/Date in our code.

So probably we should create another task for beginners to remove these calls and think about better code organization.